src/routes/Header.svelte

```diff
@@ -73,6 +74,14 @@
 		}
 		hideDropdowns();
 	}
+
+  function collapseMenu() {
+    const navBarButton = document.querySelector(`button.navbar-toggler`);
+    const navBarButtonIsDisplayed = navBarButton && window.getComputedStyle(navBarButton).display !== 'none';
+    if (navBarButtonIsDisplayed) {
+      navBarButton?.dispatchEvent(new CustomEvent('click'));
+    }
```

Comment:
This suggestion checks that the main navbar has the `show` class meaning it is open.

I also think it's more readable to use `if` instead of `&&` so I made that change in this suggestion as well.

Feel free to let me know what you think and you can use the commit suggestion button if you'd like.

Thanks!

```suggestion
    const navbarButton = document.querySelector(`button.navbar-toggler`);
    const navbarMain = document.querySelector('.navbar-collapse');
    if (navbarButton && navbarMain) {
        const navExpanded = window.getComputedStyle(navbarButton).display !== 'none' && navbarMain.classList.contains('show');
        if (navExpanded) {
            navbarButton.dispatchEvent(new CustomEvent('click'));
        }
    }
```